internal/sql/repository/chartConfig/ChartRepository.go

```diff
@@ -72,6 +107,74 @@ func NewChartRepository(dbConnection *pg.DB) *ChartRepositoryImpl {
 type ChartRepositoryImpl struct {
 	dbConnection *pg.DB
 }
+func (repositoryImpl ChartRepositoryImpl) FindBulkAppNameIsGlobal(appNameIncludes string,appNameExcludes string) ([]*App,error){
+	var apps []*App
+	err := repositoryImpl.dbConnection.
+		Model(&apps).Join("inner join charts on app.id = app_id").
+		Where("app_name like ? and app_name not like ?","%"+appNameIncludes+"%","%"+appNameExcludes+"%").
```

Comment:
use fmt.Sprintf for concatenating string eg, fmt.Sprintf("%s%s%s", "%", appNameIncludes, "%")